src/test/polymer/polymer-core-feature_test.ts

```diff
@@ -62,43 +62,62 @@ suite('PolymerCoreFeatureScanner', () => {
     const [a, b, base, invalid] = features;
 
     assert.equal(a.description, 'Feature A');
-    assert.lengthOf(a.warnings, 0);
-    assert.lengthOf(a.methods, 1);
-    assert.lengthOf(a.properties, 1);
-    assert.equal(a.methods[0].name, 'methodA');
-    assert.equal(a.methods[0].type, 'Function');
-    assert.equal(a.methods[0].description, 'Method A');
+    assert.deepEqual(a.warnings, []);
+    assert.deepEqual(
+        a.properties.map(
+            ({name, type, description}) => ({name, type, description})),
+        [{
+          name: 'propA',
+          type: 'Array',
+          description: 'Prop A',
+        }]);
+    assert.deepEqual(
+        a.methods.map(
+            ({name, type, description}) => ({name, type, description})),
+        [{
+          name: 'methodA',
+          type: 'Function',
+          description: 'Method A',
+        }]);
 
     assert.equal(b.description, 'Feature B');
-    assert.lengthOf(b.warnings, 0);
-    assert.lengthOf(b.methods, 1);
-    assert.lengthOf(b.properties, 0);
-    assert.equal(b.methods[0].name, 'methodB');
-    assert.equal(b.methods[0].type, 'Function');
+    assert.deepEqual(b.warnings, []);
+    assert.deepEqual(b.properties, []);
+    assert.deepEqual(
+        b.methods.map(
+            ({name, type, description}) => ({name, type, description})),
+        [{
+          name: 'methodB',
+          type: 'Function',
+          description: '',
+        }]);
 
     assert.equal(base.description, 'Polymer.Base declaration');
-    assert.lengthOf(base.warnings, 0);
-    assert.lengthOf(base.methods, 1);
-    assert.lengthOf(base.properties, 0);
-    assert.equal(base.methods[0].name, 'methodBase');
-    assert.equal(base.methods[0].type, 'Function');
+    assert.deepEqual(base.warnings, []);
+    assert.deepEqual(base.properties, []);
+    assert.deepEqual(
+        base.methods.map(
+            ({name, type, description}) => ({name, type, description})),
+        [{
+          name: 'methodBase',
+          type: 'Function',
+          description: '',
+        }]);
 
     assert.lengthOf(invalid.warnings, 1);
   });
 
   test('resolves the Polymer.Base class', async() => {
     const analyzer = new Analyzer({
       urlLoader: new FSUrlLoader(
-          // This directory contains 3 files copied from Polymer 1.x core.
+          // This directory contains files copied from Polymer 1.x core.
           path.resolve(__dirname, '../static/polymer-core-feature/')),
     });
 
     const analysis = await analyzer.analyzePackage();
     const features = analysis.getFeatures({id: 'Polymer.Base', kind: 'class'});
     assert.equal(features.size, 1);
     const polymerBase = features.values().next().value;
-    assert.lengthOf(polymerBase.methods, 35);
```

Comment:
It's better to have these assertions than not to, I think. Just in cases where you can reasonably do the assertion as a deepEquals that's much better.